### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dominikwilkowski**

### CODE REVIEW

**Review by dominikwilkowski — COMMENTED** [review]
test

**Review by dominikwilkowski — COMMENTED** [review]
testing

**Review by dominikwilkowski — COMMENTED** [review]
testing 3

**Review by dominikwilkowski — COMMENTED** [review]
testing 4

**Review by dominikwilkowski — COMMENTED** [review]
testing 5

**Review by dominikwilkowski — COMMENTED** [review]
testing 6

**Review by dominikwilkowski — COMMENTED** [review]
Testing 7

**Review by gwyneplaine — APPROVED** [review]

**Review by dominikwilkowski — COMMENTED** [review]
testing 9

File: README.md [hunk 1]
```diff
@@ -1 +1 @@
-# test-voting-repo
\ No newline at end of file
+# testing
```
Comment (gwyneplaine) [inline README.md:1]: Dom is a poo poo head

**Review by gwyneplaine — CHANGES_REQUESTED** [review]

**Review by dominikwilkowski — COMMENTED** [review]
test comment

**Review by dominikwilkowski — COMMENTED** [review]
Might this be it?

**Review by dominikwilkowski — COMMENTED** [review]
^ it wasn't

**Review by dominikwilkowski — COMMENTED** [review]
...

**Review by dominikwilkowski — COMMENTED** [review]
!

**Review by dominikwilkowski — COMMENTED** [review]
:)

**Review by dominikwilkowski — COMMENTED** [review]
how is this?

**Pull request merged**